src/main/java/io/cryostat/agent/FlightRecorderHelper.java

```diff
@@ -75,7 +75,9 @@ public Optional<TemplatedRecording> createRecordingWithPredefinedTemplate(
             Recording recording = new Recording(allTemplate);
             recording.setToDisk(true);
 
-            return Optional.of(new TemplatedRecording(null, recording));
+            return Optional.of(
+                    new TemplatedRecording(
+                            new ConfigurationInfo(TemplateType.TARGET, "ALL", "ALL"), recording));
```

Comment:
I think its better to have the `allTemplate` as the name.

Reply:
Let's shelve that for a separate discussion. The convention has always been `ALL` here since the very early days of the project, so changing it or adding a new alias for it is actually a bigger question and should be handled separately from this PR I think.